### PR TITLE
Feat: Cart

### PR DESCRIPTION
**Pull request by sri-rishi**
Cart:

- Cart items
- Price detail section

### CODE REVIEW

File: components/cart.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                            Remove form Cart
+                        </button>
+                        <button class="btn btn-primary-light bg-transparent">
+                            Move to Wishlist
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="line"></div>
```
Comment (pradipgharge) [inline components/cart.html:130]: you can use br tag for line break

**Review by pradipgharge — COMMENTED** [review]
Rest looks good to merge !

**Comment by swastika0015** [comment]
Looks good to me! I think it can be merged.

**Review by Sahil25061999 — APPROVED** [review]
Cant find any suggestion code looks good just on the ui part  for mobile view you can add the price details section first and then the cart items

**Comment by iprankurpandey** [comment]
br tag can be used to break between items , rest is good

**Review by iprankurpandey — APPROVED** [review]

File: components/cart.html (hunk 1 above)
Comment (sri-rishi) [reply components/cart.html:130]: this dive is a vertical line between cart-items section and price-details section , this is not for line break

Comment (sri-rishi) [reply components/cart.html:130]: This div is for vertical line between cart-items section and place order section not for line break

**Review by pradipgharge — APPROVED** [review]

**Pull request merged**